demo/admin/src/dashboard/LatestContentUpdates.tsx

```diff
@@ -1,11 +1,33 @@
-import { gql } from "@apollo/client";
-import { useTableQuery } from "@comet/admin";
+import { gql, useQuery } from "@apollo/client";
 import { LatestContentUpdatesDashboardWidget } from "@comet/cms-admin";
 import { useContentScope } from "@src/common/ContentScopeProvider";
+import { GQLLatestContentUpdatesQueryVariables } from "@src/dashboard/LatestContentUpdates.generated";
 import { categoryToUrlParam } from "@src/utils/pageTreeNodeCategoryMapping";
 import React from "react";
 
-import { GQLLatestContentUpdatesQuery, GQLLatestContentUpdatesQueryVariables } from "./LatestContentUpdates.generated";
+import { GQLLatestContentUpdatesQuery } from "./LatestContentUpdates.generated";
+
+export const LatestContentUpdates = () => {
+    const { scope } = useContentScope();
+    const { data, loading, error } = useQuery<GQLLatestContentUpdatesQuery, GQLLatestContentUpdatesQueryVariables>(LATEST_CONTENT_UPDATES_QUERY, {
+        variables: {
+            scope,
+        },
+    });
+
+    return (
+        <LatestContentUpdatesDashboardWidget<GQLLatestContentUpdatesQuery["paginatedPageTreeNodes"]["nodes"][number]>
+            data={{
+                rows: data?.paginatedPageTreeNodes.nodes,
+                loading,
+                error,
+            }}
+            getUrlFromPageTreeNode={(node) =>
+                `/${node.scope.domain}/${node.scope.language}/pages/pagetree/${categoryToUrlParam(node.category)}/${node.id}/edit`
```

Comment:
You can also do 

```
const contentScope = useContentScope();

`${contentScope.match.url}/pages/pagetree/${categoryToUrlParam(node.category)}/${node.id}/edit`
```

Reply:
```
Property 'match' does not exist on type 'ContentScope'
```

@thomasdax98 `scope.match` doesn't seem to exist 🤷🏼  
Does the demo-project maybe have an older implementation of scopes? 🤔

Reply:
That is because you're destructuring: `const { scope } = useContentScope()`. Should be `const contentScope = useContentScope()` for this to work

Reply:
Ah 😅